docs/tour/create_simple_cluster.md

```diff
@@ -0,0 +1,38 @@
+## Create a simple cluster
+
+We'll create a simple cluster
+
+Make sure you have run:
+`export KOPS_STATE_STORE=s3://clusters.example.com`
```

Comment:
Do we want to state to run these in bash or ssh? Also add the formatting love for bash highlighting